CRM/Utils/Date.php

```diff
@@ -438,6 +438,23 @@ public static function customFormat($dateString, $format = NULL, $dateParts = NU
     }
   }
 
+  /**
+   * Wrapper for customFormat that takes a timestamp
+   *
+   * @param int $timestamp
+   *   Date and time in timestamp format.
+   * @param string $format
+   *   The output format.
+   * @param array $dateParts
+   *   An array with the desired date parts.
+   *
+   * @return string
+   *   the $format-formatted $date
+   */
+  public static function customFormatTs($timestamp, $format = NULL, $dateParts = NULL) {
+    return CRM_Utils_Date::customFormat(date("Y-m-d H:i:s", $timestamp)  , $format, $dateParts);
```

Comment:
Maybe, the test is failing due to the extra space after `)`. Can you try removing that and ask jenkin to retest?

Reply:
Urgh!  Stray spaces removed.